CogniteSdk.Types/Assets/Asset.cs

```diff
@@ -96,6 +96,13 @@ public override int GetHashCode()
 
         /// <inheritdoc />
         public override string ToString() => Stringable.ToString(this);
+
+        /// <summary>
+        /// A list of labels associated with this asset
+        /// </summary>
+        [Obsolete("The Label attribute is in development, and currently only available for use in playground")]
+        public List<Label> Labels { get; set; }
```

Comment:
I think this should be IEnumerable<string>. We don't use "single case unions" for other properties so we should probably not use for labels as well.

Reply:
Thus Label.cs can be removed.